source/localizable/addons-and-dependencies/managing-dependencies.md

```diff
@@ -27,7 +27,9 @@ Thus it reloads your app if you install new dependencies via `bower install <dep
 
 ## Other assets
 
-Assets not available as an addon or Bower package should be placed in the `vendor` folder in your project.
+Third-party assets not available as an addon or Bower package should be placed in the `vendor` folder in your project.
```

Comment:
I think that this should say "Third-party JavaScript," not "Third-party assets." Are there any cases where someone would be using third-party non-JavaScript files that wouldn't be better served by being placed in `public/`?